### PR TITLE
SQL backend works

### PR DESCRIPTION
**Pull request by garrypolley**
- Updated tests.  SQL code is fully covered
- Authenticated Backend works
### Notes
# 
- Need to do mongo still
- Need to release this to PyPI

### CODE REVIEW

File: django_generic_permissions/models.py [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 # Define the location of the backends
 sql_backend = 'django_generic_permissions.sql_models'
-mogoengine_backend = 'django_generic_permissions.mongoengine_models'
+mongoengine_backend = 'django_generic_permissions.mongoengine_models'
 
-backend_module = (import_module(mogoengine_backend) if backend_db == 'mongoengine'
+backend_module = (import_module(mongoengine_backend) if backend_db == 'mongoengine'
```
Comment (estebistec) [inline django_generic_permissions/models.py:20]: This looks kinda familar :)

Comment (garrypolley) [inline django_generic_permissions/models.py:20]: You learn from the best.  :cat:

**Comment by estebistec** [comment]
So, I can't seem to make general comments on the code. Should I just go back to old pull requests?

**Comment by garrypolley** [comment]
Yeah, or just comment on the commits.

**Pull request merged**